front/app/components/VoteControl/VotingDisabled.test.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { render, screen } from 'utils/testUtils/rtl';
+import { makeUser } from 'services/__mocks__/users';
+import { getProject } from 'services/__mocks__/projects';
+
+import VotingDisabled from './VotingDisabled';
+const defaultProps = {
+  projectId: 'projectId',
+};
+
+const mockProject = getProject(
+  defaultProps.projectId,
+  'continuous',
+  'ideation'
+);
+const mockUser = makeUser();
+
+jest.mock('utils/cl-intl');
+jest.mock('hooks/useProject', () => jest.fn(() => mockProject));
+jest.mock('hooks/useAuthUser', () => jest.fn(() => mockUser));
+
+describe('VotingDisabled', () => {
+  it('renders the component', () => {
```

Comment:
This looks great! Once suggestion - check that it doesn't render the component when combinations of props tell it not to. I'm not familiar enough with the logic here but the idea would be to add the different possible combinations for when it needs to render and check that it does and the different possible combinations when it shouldn't render and check that is doesn't.

**Tip**: to check that it's not in the document use `queryByTestId('testId').not.toBeInTheDocument()`

Reply:
Added!